src/EnrollRestaurant/Confirm.tsx

```diff
@@ -0,0 +1,97 @@
+import { Grid, Typography, Chip, List, ListItem } from "@mui/material";
+import React, { FC } from "react";
+import { observer } from "mobx-react-lite";
+import store from "./store/store";
+import { FOOD_TYPES, SHARE_LIST } from "./store/enums";
+import { Box } from "@mui/system";
+
+const Confirm: FC = observer(() => {
+  const {
+    foodTypeId,
+    resName,
+    mainAddr,
+    detailAddr,
+    shareList,
+    price,
+    tagList,
+    memo,
+  } = store;
+
+  return (
+    <>
+      <Typography variant="h6" gutterBottom>
+        맛집 등록 정보 확인
+      </Typography>
+      <Grid container spacing={3}>
+        <Grid item xs={3}>
+          맛집 이름
+        </Grid>
+        <Grid item xs={9}>
+          {resName}
+        </Grid>
+        <Grid item xs={3}>
+          주소
+        </Grid>
+        <Grid item xs={9}>
+          {mainAddr}
+        </Grid>
+        <Grid item xs={3}>
+          상세 주소
+        </Grid>
+        <Grid item xs={9}>
+          {detailAddr}
+        </Grid>
+        <Grid item xs={3}>
+          음식 종류
+        </Grid>
+        <Grid item xs={9}>
+          {FOOD_TYPES[foodTypeId].name}
```

Comment:
```suggestion
          {FOOD_TYPES[foodTypeId]?.name}
```

해당 값이 없을 때 오류가 발생할 수 있을 거 같아 예외 처리 해야할 것 같습니다.